notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_online_prediction.ipynb

```diff
@@ -0,0 +1,1545 @@
+{
```

Comment:
<p>..how to use with the Vertex AI SDK for Python to set up Vertex AI Model Monitoring V2 for online predictions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>